static/js/index.js

```diff
@@ -214,13 +256,16 @@
   // We should aim to have the same result whether we are rending from here
   // or from the Go template!
   var renderHits = function(search, result) {
-
+	
+	// TabIndex starts from 9 we already got 8 elements on html page
+	var tabIndexCount = 9;
+	
     var html = "";
     for (var i = 0; i < (result["h"] || []).length; i++) {
       var hit = result["h"][i];
       html += "<div class='r'>" +
-                "<h3><a href='"+hit["u"]+"'>"+htmlSafe(hit["t"])+"</a></h3>" +
-                "<div class='u'><a href='"+hit["u"]+"'>" + simplifyURL(hit["u"]) + "</a></div>" +
+                "<h3><a href='"+hit["u"]+"' tabindex='"+(tabIndexCount+=1)+"'>"+htmlSafe(hit["t"])+"</a></h3>" +
+                "<div class='u'><a href='"+hit["u"]+"' tabindex='"+(tabIndexCount+=1)+"'>" + simplifyURL(hit["u"]) + "</a></div>" +
```

Comment:
Sorry if that wasn't clear! tabindex on the h3.a is great. But I don't think we need it on the second one (`<div class='u'>`). Each "tab" keypress should take you to the next result!